app/controllers/cart_controller.rb

```diff
@@ -1,9 +1,11 @@
 class CartController < ApplicationController
   def create
-    if Cart.last
-      cart = Cart.last
+    if session[:cart_id]
+      cart = Cart.find(session[:cart_id])
+    #get cart from database
```

Comment:
Is the comment necessary? If it is then please set the comment above the code that the comment is explaining.